lang/en/lang.php

```diff
@@ -11,6 +11,12 @@
         'page_name' => 'Topic Page',
         'page_help' => 'Page name to use for clicking on a conversation topic.'
     ],
+    'posts' => [
+        'component_name' => 'Posts List',
+        'component_description' => 'Displays a list of all posts.',
+        'per_page' => 'Posts per page',
+        'per_page_validation' => 'Invalid format of the posts per page value'
```

Comment:
@Inter-Net-Pro would you mind changing the language of the `per_page_validation` key to say the following:
```suggestion
        'per_page_validation' => 'Posts per page must be a number'
```
And update the Russian translation to the same?

Reply:
Yes. Corrected.